wiki/Setup.md

```diff
@@ -8,29 +8,58 @@ Make sure Chrome or Chromium is installed
 
 ## Authentication:
 
-You must define the session token in the config:
+You must define the session token or (email and password) for Microsoft Login in the config:
 
-You can find the session token manually from your browser:
+- ### Session Token Authentication:
 
-1. Go to `https://chat.openai.com/api/auth/session`
-2. Press `F12` to open console
-3. Go to `Application` > `Cookies`
-4. Copy the session token value in `__Secure-next-auth.session-token`
-5. Paste it into `config.json` in the current working directory
+  You can find the session token manually from your browser:
 
-```json
-{ "session_token": "<YOUR_TOKEN>" }
-```
+  1. Go to `https://chat.openai.com/api/auth/session`
+  2. Press `F12` to open console
+  3. Go to `Application` > `Cookies`
+  4. Copy the session token value in `__Secure-next-auth.session-token`
+  5. Paste it into `config.json` in the current working directory
+
+  ```json
+  { "session_token": "<YOUR_TOKEN>" }
+  ```
+
+- ### Email/Password Login Authentication:
 
-You can use `Xvfb` to emulate a desktop environment. It should automatically get the `cf_clearance` given no captcha.
+  ```json
+  { "email": "<EMAIL>", 
+    "password": "<PASSWORD>",
+    "captcha": "<2CAPTCHA_API_KEY>"
+    }
+  ```
+  **Note: 2Captcha is required for Email/Password Login**
+- ### Microsoft Login Authentication:
 
-Search it up if you don't know. Ask ChatGPT.
+  ```json
+  { "email": "<EMAIL>", 
+    "password": "<PASSWORD>",
+    "isMicrosoftLogin": True
```

Comment:
is it True or true?

Reply:
it's set to False by default in the init Function of the class, only when isMicrosoftLogin is set to True in the config then the script will commence with the Microsoft Login

Reply:
oh I misunderstood the question, I think it should be True right?

Reply:
No. In Python it's True. In JSON, it's true. (I think)

Reply:
let me test give me a moment

Reply:
![image](https://user-images.githubusercontent.com/82155626/209752105-0f84aed8-6bd0-496b-a25c-a0da2fe12e57.png)
well when defining the config as a python dict it should be True but when you're importing from an external JSON file I believe it should be true. I will test if external JSON file works without modifying the code.

Reply:
It seems there is a difference between JSON and Python dict. Will need to document that (even though most people can figure it out)

Reply:
Tested it locally, `json.loads` converts `true` to `True` automatically when importing a json file so it's just a matter of documenting the wiki/readme properly.